ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -124,6 +124,42 @@ objects:
 
         mkdir -p "${HOME}"
 
+        # Install python-openstackclient which is currently required to run
+        # some tests (in parallel mode). See: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/drivers/in_tree.go#L956
+        cat > /etc/yum.repos.d/rdo-stein.repo <<-"EOF_CAT"
+        [openstack-stein]
+        name=OpenStack Stein Repository
+        baseurl=http://mirror.centos.org/centos/7/cloud/$basearch/openstack-stein/
+        gpgcheck=1
+        enabled=1
+        gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Cloud
+        EOF_CAT
+
+        cat > /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Cloud <<-"EOF_CAT"
```

Comment:
nit: `<<EOF` (or `EOF_CAT`, if you want) is sufficient, because you don't use the tab-indents that `<<-` enables.

Reply:
Ack, I can remove the quoted EOF_CAT's